Dolapo47/Auto-Mart
# Ft updateorder ui #167200549

#### What does this PR do?
- A summary of what this task is about
This PR creates the update orders page for users
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-created the desktop view for users
-created the mobile view for users
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
navigate to the update order page
check the regular desktop view and the mobile view using the developer console
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#167200549](https://www.pivotaltracker.com/n/projects/2346179)`)
